handbook/company/pricing-features-table.yml

```diff
@@ -33,6 +33,12 @@
     - name: Update apps on macOS computers
       tier: Premium
       comingSoon: true
+    - name: Map macOS settings to computers with Puppet module
+      tier: Premium
+      comingSoon: false
+    - name: Reduce gap in coverage with end user initiated migration
```

Comment:
Small copy point: shoudl be "end user-initiated migration"

medium point: "gap in coverage" is also a term that I made up, I'm not sure this'll mean anything to the audience. 

Larger point: Not sure I like this wording. When I hear "end user-initiated" I think, "we're relying on them to do it. They're never gonna do it" which is the opposite of reducing gap in coverage. I think the point is not that end users are initiating migration (that has to happen always), it's that we built a nagging system. "Migration reminders" "migration nags" "migration nudges" "Nudge for migrations"?

Reply:
Good ideas.  Agreed it could be shorter.

Reply:
Yeah, I was trying to sell the value: gap in coverage. Agreed this could be confusing.

I like "Nudge for migrations." Changed it to this.